index.html

```diff
@@ -126,11 +126,6 @@ <h3 style="margin-top: 0px; font-weight: bold;">Updates</h3>
 {% assign anchor_name="schedule" %}
 {% assign anchor_html_entity="h3" %}
 {% include anchor.html %}
-<div class="alert alert-warning">
-  <span class="glyphicon glyphicon-warning-sign"></span>
-  As this is a new class, the syllabus is subject to change in response to
-  feedback and class needs throughout the term.
-</div>
```

Comment:
While dropping the "As this is a new class" clause is good, I'd leave the rest -- always good to have a little wiggle room :)